main.js

```diff
@@ -0,0 +1,45 @@
+const menuBtn = document.querySelector('.navbarmenu');
+const menuItems = document.querySelector('.itemslist');
+const listItems = document.querySelectorAll('.listitem');
+const navBar = document.querySelector('.navbar');
+const navbarIcon = document.querySelector('.navbaricon');
+const crossButton = document.querySelectorAll('.btn-line');
+
```

Comment:
- [ ] Great work. you are missing Eslint checks. ESLint is a tool for identifying and reporting on patterns found in your JavaScript code, with the goal of making it more consistent and avoiding bugs. you can check how to add it [here](https://github.com/microverseinc/linters-config/tree/master/javascript).